internal/helmupdater/helmupdater.go

```diff
@@ -0,0 +1,64 @@
+package helmupdater
+
+import (
+	"os"
+	"path/filepath"
+
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/downloader"
+	"helm.sh/helm/v3/pkg/getter"
+	"helm.sh/helm/v3/pkg/registry"
+)
+
+const (
+	ChartYaml = "Chart.yaml"
+)
+
+func FoundChartYamlInDirectory(path string) bool {
```

Comment:
Sometimes I tend to let the code self explain because documentation and comments need also maintenance when changing the code they're documenting or commenting, but sure, no problem... I've added the doc.